core/src/main/scala/cats/instances/future.scala

```diff
@@ -8,12 +8,18 @@ import scala.concurrent.{ExecutionContext, Future}
 
 trait FutureInstances extends FutureInstances1 {
 
-  implicit def catsStdInstancesForFuture(implicit ec: ExecutionContext): MonadError[Future, Throwable] with CoflatMap[Future] =
-    new FutureCoflatMap with MonadError[Future, Throwable]{
+  implicit def catsStdInstancesForFuture(implicit ec: ExecutionContext): MonadError[Future, Throwable] with CoflatMap[Future] with MonadRec[Future] =
+    new FutureCoflatMap with MonadError[Future, Throwable] with MonadRec[Future] {
       def pure[A](x: A): Future[A] = Future.successful(x)
 
       def flatMap[A, B](fa: Future[A])(f: A => Future[B]): Future[B] = fa.flatMap(f)
 
+      final def tailRecM[B, C](b: B)(f: B => Future[(B Xor C)]): Future[C] =
```

Comment:
@johnynek I'm pretty sure that was only an issue with the original Akka `Future` implementation, and that by the time SIP-14 landed `flatMap` was stack-safe (thanks to pressure from Twitter and inspired by `c.t.u.Future`). I'll double-check that, though.